File: BackendAPI/DataLayer/Models/Entities/answer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DataLayer.Models.Entities
{
    public class answer
    {
        public int id { get; set; }
        public int? user_id { get; set; }
        public int? question_id { get; set; }
        public string? selected_option { get; set; }
        public bool? is_correct { get; set; }
        public string? quiz_reference { get; set; }
    }
}

File: Frontend/src/models/answer.ts
export class Answer {
  id: number | null;
  user_id: number | null;
  question_id: number | null;
  user_name: string | null;
  question: string | null;
  selected_option: string | null;
  is_correct: boolean | null;
  quiz_reference: string | null;
}

export class QuizSummary {
  quiz_reference: number | null;
  total_question: number | null;
  correct: number | null;
  wrong: number | null;
}
